pytermgui/ansi_interface.py

```diff
@@ -2,27 +2,36 @@
 Various functions to interface with the terminal, using ANSI sequences.
 
 Credits:
-
 - https://wiki.bash-hackers.org/scripting/terminalcodes
 - https://gist.github.com/fnky/458719343aabd01cfb17a3a4f7296797
 """
 
 from __future__ import annotations
 
+import asyncio
```

Comment:
Not really sure on the usage of `async` functionality for this one use, as all other parts of the library use threads instead.

Reply:
It is for `WinSIGWINCH`